components/src/molecules/product-list-item/ProductListItem.html

```diff
@@ -0,0 +1,114 @@
+<a-list-item tag="div" class="a-product-list-item">
+  <div class="a-product-list-item__image">
+    <!--@slot Custom product image (Named slots)-->
+    <slot name="image">
+      <a-image
+        :src="imageUrl"
+        :alt="name"
+      />
+    </slot>
+  </div>
+  <div class="a-product-list-item__info">
+    <div class="a-product-list-item__data">
+      <!--@slot Custom product name (Named slots)-->
+      <slot name="name">
+        <router-link
+          :to="url"
+          class="link link--inverted a-product-list-item__name"
+        >
+          {{ name }}
+        </router-link>
+      </slot>
+      <!--@slot Custom product price (Named slots)-->
+      <slot name="price">
+        <a-price
+          :price="price"
+          :special-price="specialPrice"
+          :old-price="oldPrice"
+          class="a-product-list-item__price-wrapper"
+          old-price-class="a-product-list-item__price-wrapper--old"
+          special-price-class="a-product-list-item__price-wrapper--special"
+        />
+      </slot>
+
+      <a-list
+        v-if="options"
+        tag="dl"
+        class="list--description a-product-list-item__attributes"
+      >
+        <template v-for="item in options">
+          <!-- @slot Custom product item option (Scoped slot) -->
+          <slot
+            name="option"
+            :option="item"
+          >
+            <a-list-item
+              tag="dt"
+              class="list--description__label"
+            >
+              {{ item.label }}
+            </a-list-item>
+            <a-list-item
+              tag="dd"
+              class="list--description__value"
+            >
+              {{ item.value }}
+            </a-list-item>
+          </slot>
+        </template>
+      </a-list>
+    </div>
+
+    <div
+      :class="[
+        'a-product-list-item__actions',
+        { 'a-product-list-item__actions--to-right': hideQty }
+      ]"
+    >
+      <div
+        v-if="!hideQty"
+        class="a-product-list-item__qty-wrapper"
+      >
+        <!--@slot Custom product quantity (Named slots)-->
+        <slot name="quantity">
+          <a-quantity-update
+            :value="qty"
+            input-id="qty_1"
+            class="product-list-item__qty"
+            @update="onChange"
+            decrement-aria-label="Decrease the quantity"
+            decrement-icon-title="Minus mark"
+            increment-aria-label="Increase the quantity"
+            increment-icon-title="Plus mark"
+          >
+            <template #label>
+              <a-label
+                class="product-list-item__qty-label"
+                for="qty_1"
+              >
+                Qty:
+              </a-label>
+            </template>
+          </a-quantity-update>
+        </slot>
+      </div>
+
+      <div class="a-product-list-item__action-buttons-wrapper">
+        <!--@slot Custom product remove button (Named slots)-->
+        <slot name="removeAction">
```

Comment:
Change to scoped slot with `onRemove`.
After all the above you can update `With slot` story where `<template>` is used and change the values to these from scope instead of defining them 2 times